resources/views/users/view.blade.php

```diff
@@ -645,6 +645,7 @@
                        </div>
                        <div class="col-md-9">
                            {{Helper::formatCurrencyOutput($user->getUserTotalCost()->total_user_cost)}}
+                           @if($user->getUserTotalCost()->total_user_cost > 0)
```

Comment:
Wouldn't we want to put this if/else a little higher up so we don't display the total cost or even the row if there is no total cost to provide?